File: DialogLibrary/src/main/java/org/caojun/dialog/RandomUtils.java
package org.caojun.dialog;

class RandomUtils {
    static int getRandom(int min, int max) {
        return (int)(Math.random() * (max + 1 - min) + min);
    }

    static boolean getRandom() {
        return getRandom(0, 1) == 0;
    }
}

File: DiceLibrary/src/main/java/org/caojun/dice/RandomUtils.kt
package org.caojun.dice

/**
 * Created by CaoJun on 2017/8/15.
 */
object RandomUtils {
    fun getRandom(min: Int, max: Int): Int {
        return (Math.random() * (max + 1 - min) + min).toInt()
    }
}